### ISSUE TITLE
Check and correct activation functions

### ISSUE THREAD

**Issue by JakeRoggenbuck** [jabacat/jml#49]
Correct activation functions

- [ ] List activation functions
- [ ] Check all listed functions

Make a graph of each function and display it

**Comment by adamhutchings** [comment]
Is this issue still open? Or did it get resolved?

**Comment by JakeRoggenbuck** [comment]
I can double check every function. Should they all be through the origin?

**Comment by adamhutchings** [comment]
No, not necessarily. The sigmoid as it is normally defined is 0.5 at x=0, for example.
